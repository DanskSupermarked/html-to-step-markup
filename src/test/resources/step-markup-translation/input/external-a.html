Granit: plejemiddel<br><a href="http://www.bilka.dk/Bolig/Moebler/moebelpleje">Se vores sortiment af mÃ¸belpleje her</a>
