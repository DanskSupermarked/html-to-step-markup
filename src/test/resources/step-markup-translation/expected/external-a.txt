Granit: plejemiddel<return/>
Se vores sortiment af mÃ¸belpleje her www.bilka.dk/Bolig/Moebler/moebelpleje
